### PR TITLE
Added MaaS integration data in services model

### PR DESCRIPTION
**Pull request by benethor**

### CODE REVIEW

File: src/selena/services/models.py [hunk 1]
```diff
@@ -159,6 +159,15 @@ class Service(models.Model):
         choices=AuthMethodChoices(),
         default=AuthMethodChoices.none.id,
     )
+    maas_intkey = models.CharField(
```
Comment (h00dy) [inline src/selena/services/models.py]: Maybe integration_key would be a better name. People may not know what is maas.

File: src/selena/selena/settings.py [hunk 2]
```diff
@@ -124,6 +124,11 @@
 LOGIN_URL = '/login/'
 LOGIN_REDIRECT_URL = '/'
 
+ACTIONEAR_NOTIFICATIONS_CLIENT = 'selena'
```
Comment (deejay1) [inline src/selena/selena/settings.py]: drop it from here please

File: src/selena/services/models.py (hunk 1 above)
Comment (deejay1) [inline src/selena/services/models.py]: +1

File: src/selena/services/models.py [hunk 3]
```diff
@@ -159,6 +159,15 @@ class Service(models.Model):
...
+        _('MaaS Integration Key'),
+        max_length=32,
+        blank=True,
+        null=True
+    )
+    sc_id = models.PositiveIntegerField(
+        _('Service Catalog ID'), blank=True, null=True
```
Comment (deejay1) [inline src/selena/services/models.py:169]: Catalogue :P

Comment (benethor) [inline src/selena/services/models.py:169]: again, SC uses US English form, let's keep it consistent throughout the ecosystem :P

**Comment by deejay1** [comment]
:+1:

**Comment by ch40s3k** [comment]
:thumbsup:

**Pull request merged**